### PR TITLE
[ULMS-1740] Add migrations

### PR DESCRIPTION
**Pull request by mgrachev**

### CODE REVIEW

**Review by khodzha — APPROVED** [review]

File: migrations/20220204103437_create_table_session_history.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS session_history
+(
+    uuid         uuid,
+    agent_id     agent_id,
+    classroom_id uuid,
+    time_range   tsrange
```
Comment (manifest) [inline migrations/20220204103437_create_table_session_history.sql:6]:
```
tstzrange
```

File: migrations/20220204101411_create_table_session.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS session
+(
+    uuid         uuid,
+    agent_id     agent_id,
+    classroom_id uuid,
+    replica_id   varchar,
```
Comment (manifest) [inline migrations/20220204101411_create_table_session.sql:6]:
```
text
```

File: migrations/20220204101411_create_table_session.sql [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+    started_at   timestamp
```
Comment (manifest) [inline migrations/20220204101411_create_table_session.sql:7]:
```
timestamptz
```

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,5 @@
+deps:
```
Comment (manifest) [inline Makefile:1]: .PHONY: deps db

File: migrations/20220204103437_create_table_session_history.sql (hunk 1 above)
Comment (mgrachev) [reply migrations/20220204103437_create_table_session_history.sql:6]: Done.

File: migrations/20220204101411_create_table_session.sql (hunk 2 above)
Comment (mgrachev) [reply migrations/20220204101411_create_table_session.sql]: Done.

File: migrations/20220204101411_create_table_session.sql (hunk 3 above)
Comment (mgrachev) [reply migrations/20220204101411_create_table_session.sql]: Done.

File: Makefile (hunk 4 above)
Comment (mgrachev) [reply Makefile:3]: Done.

**Review by manifest — APPROVED** [review]

**Pull request merged**